src/BlackSlope.Api.Common/stylecop.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
+  "settings": {
+    "orderingRules": {
+      "usingDirectivesPlacement": "outsideNamespace"
+    },
+    "layoutRules": {
+      "newlineAtEndOfFile": "require"
+    },
+    "documentationRules": {
+      "documentExposedElements": false, // Enable this to enforce documenting public elements

```

Comment:
This doesn't break the JSON parser? comments aren't part of the JSON spec

Reply:
It doesn't seem to break StyleCop, but I didn't realize it broke JSON spec. Fixing that now.

Reply:
Personally I don't think this would be an issue since JSON for configuration notation is only meant for the target application, it should be fine as long as the target application is fine with it. Which I believe in this case styleCop is. I put comments in appsettings.json once in a while. But for this particular case, I'm not sure if this comment is really needed, I guess code is clear enough by itself.